### PR TITLE
mallet 2.0.8

### PR DESCRIPTION
**Pull request by imgarylai**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [ ] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

File: mallet.rb [hunk 1]
```diff
@@ -3,13 +3,13 @@ class Mallet < Formula
...
+  url "http://mallet.cs.umass.edu/dist/mallet-2.0.8.tar.gz"
+  sha256 "5b2d6fb9bcf600b1836b09881821a6781dd45a7d3032e61d7500d027a5b34faf"
   head "https://github.com/mimno/Mallet.git"
 
   devel do
-    url "http://mallet.cs.umass.edu/dist/mallet-2.0.8RC3.tar.gz"
-    sha256 "cced45641a671c41ef63d498ce39ddcb9cd904fb6da2f6ca0fd4488b7e4ee4fc"
+    url "https://github.com/mimno/Mallet/archive/v2.0.8RC3.tar.gz"
+    sha256 "e7ffcb4d9cc936e9c66d484970b79774c47c18f7133bd6c99fab10530feef5ab"
   end
```
Comment (jonchang) [inline mallet.rb]: I don't think this `devel` block is needed anymore (2.0.8rc3 would be older than 2.0.8).

Comment (imgarylai) [inline mallet.rb]: Agree. On Umass website, it only shows 2.0.8 now. How about I remove it in my next update?

**Comment by imgarylai** [comment]
@jonchang Changes updated.

**Comment by iMichka** [comment]
@imgarylai Thanks for the changes, and thanks for your contribution!

**Pull request closed without merging**